### ISSUE TITLE
How stock exchange make money?

### ISSUE THREAD

**Issue by vub** [vub/TIL#93]
https://www.investopedia.com/articles/investing/050515/how-nyse-makes-money.asp

https://vi.wikipedia.org/wiki/Gi%C3%A1m_%C4%91%E1%BB%91c_c%C3%B4ng_ngh%E1%BB%87_th%C3%B4ng_tin

**Comment by vub** [comment]
How The NYSE Makes Money
FACEBOOK
TWITTER
LINKEDIN
By SHOBHIT SETH
 Updated Mar 31, 2020
The New York Stock Exchange (NYSE) is one of the world’s largest stock exchanges. Stock exchanges allow investors and traders to make money by providing them a marketplace for trading securities. They also allow companies to raise money by listing different kinds of securities. For providing such services and marketplace, exchanges collect transaction fees from market participants and companies. Exchanges also offer various products and services used for trading and related activities.

This article explores the various sources of revenue and income for the NYSE.

Transaction Fee Revenue
People come to the NYSE as it maintains an efficient marketplace with fair price discovery and ensures sufficient liquidity in the market. The NYSE charges fees in various forms to these market participants. Each trade that occurs on the NYSE attracts a transaction fee from the trading parties. All trades occur through registered market participants, including brokerage firms, trading houses, and asset management companies. In addition to the transaction fee, these participants also pay a one-time registration fee and a recurring annual membership fee to the NYSE.

Listing Fee Revenue
Companies that are in need of capital can raise money by listing their securities on the NYSE after meeting eligibility criteria. They need to pay a one-time listing fee and then a recurring annual fee for listing and trading services on the NYSE platform. Equities remain the most common securities listed, but the NYSE allows other instruments to be listed, including preferred stocks, warrants, bonds, ETPs, funds, options, structured products, capital securities, mandatory convertibles, and repackaged securities.

The initial one-time listing fee is usually based on the total number of shares to be listed through the issue. After that, the NYSE charges fees based on corporate actions, like the issuance of additional shares through rights issue, bonus issue, stock split, etc.

At present, there are more than 2,400 companies listed on the NYSE. In 2014, 195 issuers listed their securities on the NYSE, which helped them raise more than $183 billion. That includes 129 initial public offerings (IPOs) that raised $70.3 billion.

Data Fee Revenue
Market data—real-time data, historical data, summary data, and reference data—constitute a large portion of revenue earnings for the NYSE. Market participants need historical data for research and analysis, real-time data for ongoing trading and investment activities, summary data for reporting and auditing, and reference data for security-specific details like symbols and corporate actions.

Based on its proprietary data, the NYSE offers all such data through various data feeds, end-of-day reports, and data software products. For example, a trader willing to backtest his newly developed high-frequency trading algorithm would need a NYSE data feed, while a researcher, checking past performance of a NYSE-listed stock when it declared a dividend, would like to check the historical data.

Full information regarding NYSE data services and data products pricing can be found here.

Trading Software and Technology Revenue
The NYSE offers its various technology services and trading software to large institutional clients like mutual funds and asset management companies. Such businesses require faster data and faster trade execution through dedicated products and services, which are offered by the NYSE trading software and technology product suite. This also includes co-location, where the computers of a large trading firm are placed and managed by the NYSE on NYSE premises, offering dedicated services with close proximity to the marketplace for faster trade execution and access.

Registration and Regulatory Fee Revenue
NYSE market participants, market makers, and brokers need to register and pay the registration and regulatory fee for their NYSE membership. The NYSE also charges for facilitating trading licenses. All such charges include the one-time registration fee and recurring annual charges.

NYSE Governance Services Revenue
The NYSE also offers corporate governance, risk, and compliance services to its diversified customer base.

The latest NYSE price list for its various services can be accessed here.

The NYSE was acquired by the Intercontinental Exchange, Inc. (ICE) on Nov. 13, 2013. Since them NYSE-specific revenue data are unavailable, as ICE annual reports now consolidate all data across multiple exchanges of the ICE group. The NYSE annual report for 2012, the year prior to the acquisition, indicates healthy revenue numbers from the different streams of business (source: 2012 annual report):

Image 1
Image by Sabrina Jiang © Investopedia 2020
Transaction fees constituted 63% of total revenues, with listing fees 12%, market data 9.2%, and the rest coming from technology and other sectors.

The Bottom Line
The business of stock exchanges is apparently a profitable one, as indicated by the healthy historical revenue and income details of leading stock exchanges like the NYSE. To continue its lead as the world’s top stock exchange, the NYSE will need to remain innovative by offering new products and services across its different streams. Investors willing to invest in the stocks of such marketplaces should keep a close eye on market developments and consolidation.
